### PR TITLE
JIT README: How to install LLVM on Fedora Linux

### PR DESCRIPTION
**Pull request by hroncok**
cc @brandtbucher

### CODE REVIEW

**Comment by hroncok** [comment]
As a side note, I've also inquired about having LLVM 18 available in Fedora Linux 39.

**Comment by Eclips4** [comment]
Miro, is this really should target 3.13 branch? I guess you have chose this one accidentally.

**Comment by hroncok** [comment]
> Miro, is this really should target 3.13 branch? I guess you have chose this one accidentally. For me, it looks like it should target the current main branch (3.14), and it would be nice to backport it to 3.13, since JIT is available in 3.13.

Nah, I was doing this on my phone from GitHub web edit and I managed to click the edit button on the 3.13 branch, sorry about that.

Now I belive I fixed this, but at the same time, GitHub decided to spam everybody. I am so sorry :cry:

**Comment by Eclips4** [comment]
> > Miro, is this really should target 3.13 branch? I guess you have chose this one accidentally. For me, it looks like it should target the current main branch (3.14), and it would be nice to backport it to 3.13, since JIT is available in 3.13.
> 
> Nah, I was doing this on my phone from GitHub web edit and I managed to click the edit button on the 3.13 branch, sorry about that.
> 
> Now I belive I fixed this, but at the same time, GitHub decided to spam everybody. I am so sorry 😢

No worries! Bad things happen. I've dismissed requests for review from everyone except Brandt.

**Comment by Eclips4** [comment]
However, I'm wondering why changing "README.md" triggers our CI/CD actions, which obviously should not be run for this kind of change?

**Review by savannahostrowski — APPROVED** [review]
LGTM - thanks for adding this!

**Review by ericsnowcurrently — APPROVED** [review]
LGTM

File: Tools/jit/README.md [hunk 1]
```diff
@@ -21,6 +21,12 @@ chmod +x llvm.sh
 sudo ./llvm.sh 18
 ```
 
+Install LLVM 18 on Fedora Linux 40 or newer:
+
+```
```
Comment (erlend-aasland) [inline Tools/jit/README.md]:
```suggestion
```sh
```

**Comment by hugovk** [comment]
> This could be easily fixed by changing the `.github/workflows/jit.yml`:

Could also ignore `Tools/jit/mypy.ini`

File: Tools/jit/README.md [hunk 2]
```diff
@@ -21,6 +21,12 @@ chmod +x llvm.sh
 sudo ./llvm.sh 18
 ```
 
+Install LLVM 18 on Fedora Linux 40 or newer:
+
+```sh
+sudo dnf install 'clang(major) = 18' 'llvm(major) = 18'
```
Comment (brandtbucher) [inline Tools/jit/README.md:27]: Thanks for this! Just double-checking, is the separate Clang install necessary (it's not bundled with the LLVM install)?

**Review by brandtbucher — APPROVED** [review]

File: Tools/jit/README.md (hunk 2 above)
Comment (hroncok) [reply Tools/jit/README.md:27]: Correct. Installing just LLVM does not install clang. Neither the other way around.

**Comment by brandtbucher** [comment]
Also, if somebody wants to open a PR to ignore JIT CI on the following files, go ahead:

- `Tools/jit/README.md`
- `Tools/jit/mypy.ini`
- `Python/perf_jit_trampoline.c`

**Comment by savannahostrowski** [comment]
Happy to do that! @brandtbucher

**Pull request merged**